src/stream-tcp.h

```diff
@@ -188,7 +188,6 @@ TmEcode StreamTcp (ThreadVars *, Packet *, void *, PacketQueueNoLock *);
 uint8_t StreamNeedsReassembly(const TcpSession *ssn, uint8_t direction);
 TmEcode StreamTcpThreadInit(ThreadVars *, void *, void **);
 TmEcode StreamTcpThreadDeinit(ThreadVars *tv, void *data);
-void StreamTcpRegisterTests (void);
```

Comment:
nit: Update copyright year in this source module.

Reply:
Good call! As I'm working with that one for a different ticket, I'll do it there :P